work-tracker-servlet/src/main/java/com/deere/isg/worktracker/servlet/WorkSummary.java

```diff
@@ -107,7 +107,9 @@ public String getAcceptHeader() {
     }
 
     public void setAcceptHeader(String acceptHeader) {
-        this.acceptHeader = acceptHeader;
+        if (acceptHeader != null){
+            this.acceptHeader = acceptHeader.replaceAll(";\\s*", "; ");
```

Comment:
This should replace it with ";&#8203;" instead.  This does not add any space in the visualization, but it does allow it to break to a new line there.